src/community/infix-abstract.ts

```diff
@@ -0,0 +1,110 @@
+import { Notation } from "../notation";
+import type { DecimalSource } from "break_infinity.js";
+import Decimal from "break_infinity.js";
+import { fixMantissaOverflow } from "../utils";
+
+export abstract class AbstractInfixNotation extends Notation {
+  public name = "Abstract Infix";
+
+  protected groupDigits = 3;
+
+  protected canHandleZeroExponent = true;
```

Comment:
Codacy found an issue: [expected member-variable-declaration: 'canHandleZeroExponent' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)